### DISCUSSION TITLE
Error in Apollo Vite version

### DISCUSSION THREAD

**Discussion by aldd-cangulor** [primefaces/community#1151] [PrimeVue Templates]
The them apollo vite version does not look good, the icons appear displaced.

![Screenshot_1](https://github.com/primefaces/community/assets/19193087/78b27f5f-fb8b-4de0-8c14-5967f084bbd0)
![Screenshot_2](https://github.com/primefaces/community/assets/19193087/8405e727-a18b-4043-a291-71d71a576a9d)
![Screenshot_6](https://github.com/primefaces/community/assets/19193087/f2bbcc22-98e1-4a9a-a7c0-fb7d6082cac9)

**Comment by FlexiProject** [comment]
Hi, 
We have the same problem, sakai theme works fine, but Apollo is not working as expected: 
![image](https://github.com/primefaces/community/assets/159525491/2ae6677d-0819-420f-8215-155ad95bbc4f)

**Comment by ogzcode** [comment]
Hello
We are experiencing the same problem with the Diamond theme.
![Ekran görüntüsü 2024-02-09 175714](https://github.com/primefaces/community/assets/58819819/3c4139c6-e2c3-400b-96ba-5049ace1a4f2)

**Comment by iozcelik** [comment]
Same problem here.

**Comment by FlexiProject** [comment]
Hi, 
I was able to fix this by changing primevue version. The last version where everything is working is 3.47.2, look like 3.48 breaks it.

**Reply by dcarapic** [reply]
I have the same problem on Poseidon theme. Sadly changing to 3.47.2 does not fix it. What is your primeflex and primeicons version?

**Reply by FlexiProject** [reply]
@dcarapic Check your changelog file, there you can find information what version of primevue you should use. 
This is info from Apollo theme: 
![image](https://github.com/primefaces/community/assets/159525491/5beb96e3-374e-4363-a42e-29aa30424b08)

**Comment by dsschneidermann** [comment]
This worked for me, pin all the primevue dependencies (and consider if using the template is worth it). Dependencies:
```
        "primeflex": "3.3.1",
        "primeicons": "6.0.1",
        "primevue": "3.38.1",
```
Devdependencies:
```
        "nuxt-primevue": "0.1.0"
```
